app/components/DepartureRow.tsx

```diff
@@ -7,34 +7,16 @@ function DepartureRow(props: any) {
     const {departure} = props    
 
     return(
-        <div className="departure-row">
+        departure
+        ? (<div className="departure-row">
```

Comment:
You should check for departure.departureDateTime && departure.headsign (mandatory for your user) before rendering the children comps